src/craig_bampton.jl

```diff
@@ -20,19 +20,22 @@ function craig_bampton(K, M, r, l, n)
     X1[abs.(X1) .< eps_] = 0
     X = X1[:,1:n]
     V = X1'*Kll*X1
+    V[abs.(V) .< eps_] = 0
     Kmm = X'*Kll*X
     Kmm[abs.(Kmm) .< eps_] = 0
-    Kbb = Krr + Krl*(-X1*inv(V)*X1'*Klr)
+    b = -X*inv(Kmm)*X'*Klr
```

Comment:
maybe better solution is to make own function to make these modifications, i.e.
```julia
function droptol!(A::Matrix, tol=1.0e-6)
    A[abs.(A) <. tol] = 0
    return A
end
```

and use that in your function, i.e.
```
droptol!(V)
droptol!(b)
...
```
and so on. I'm not so sure we even need this kind of manipulations at last